pageserver/src/tenant/timeline/compaction.rs

```diff
@@ -959,15 +963,16 @@ impl Timeline {
     /// the GC horizon without considering retain_lsns. Then, it does a full compaction over all these delta
     /// layers and image layers, which generates image layers on the gc horizon, drop deltas below gc horizon,
     /// and create delta layers with all deltas >= gc horizon.
-    #[cfg(test)]
     pub(crate) async fn compact_with_gc(
         self: &Arc<Self>,
         _cancel: &CancellationToken,
         ctx: &RequestContext,
     ) -> Result<(), CompactionError> {
+        use crate::tenant::storage_layer::ValueReconstructState;
         use std::collections::BTreeSet;
 
-        use crate::tenant::storage_layer::ValueReconstructState;
+        info!("running enhanced gc bottom-most compaction");
```

Comment:
add another line that logs something when we're done, using `scopeguard::defer!{}`.

Reply:
resolved in https://github.com/neondatabase/neon/pull/8103/commits/4cc18325ab1c44ddc541475a0ec92a22668981a4